clang/lib/CodeGen/CGBuiltin.cpp

```diff
@@ -18709,6 +18709,25 @@ Value *CodeGenFunction::EmitHLSLBuiltinExpr(unsigned BuiltinID,
         CGM.getHLSLRuntime().getNormalizeIntrinsic(), ArrayRef<Value *>{X},
         nullptr, "hlsl.normalize");
   }
+  case Builtin::BI__builtin_hlsl_elementwise_fmod: {
```

Comment:
delete lines 18712 to 18730
instead see
https://github.com/llvm/llvm-project/blob/bc54e5636f2080e6a35ec201d5963a2c455fe5f5/clang/lib/CodeGen/CGBuiltin.cpp#L2837-L2851

You just need to add `case Builtin::BI__builtin_elementwise_fmod` to this

Reply:
Done.